modules/Hoot/managers/layerManager.js

```diff
@@ -377,6 +390,15 @@ export default class Layers {
         sheets.insertRule( 'g.point.tag-hoot-' + mapId + ' .stroke { fill:' + color + '}', sheets.cssRules.length - 1 );
     }
 
+    setTopLayer( mapId ) {
+        this.topLayer = mapId;
+        this.hoot.context.flush();
```

Comment:
is this what you call to get the additions you put in the lines.js file to occur?

Reply:
correct. ive seen this flush get called and it always seems like a 'ok initialize the ui with this newly edited/added info'